dynamic-layers/multimedia-layer/recipes-multimedia/libcamera/libcamera.bbappend

```diff
@@ -1,2 +1,3 @@
-PACKAGECONFIG[raspberrypi] = "-Dpipelines=raspberrypi"
+SRCREV = "ed591e705c451d0ce14988ae96829a31a2ae2f9a"
```

Comment:
This should be split into another commit with the rationale of its change.

Reply:
Will do - we need to build a version of libcamera known to work with libcamera-apps and the kernel. In the past, I've run into issue where libcamera is not compatible with kernel and libcamera-apps.

Reply:
I'lll try to build libcamera-apps with upstream version of libcamera. I suspect the build won't work but let's see.

Reply:
As predicted, the build fails. We could chase a version of libcamera-apps that does build, or submit a patch upstream to bump libcamera version. I'm inclined to do the latter.

```log
| In file included from /home/yoctouser/amora/build/tmp/work/arm1176jzfshf-vfp-poky-linux-musleabi/rpi-libcamera-apps/git-r0/git/encoder/null_encoder.cpp:12:
| In file included from /home/yoctouser/amora/build/tmp/work/arm1176jzfshf-vfp-poky-linux-musleabi/rpi-libcamera-apps/git-r0/git/encoder/null_encoder.hpp:15:
| In file included from /home/yoctouser/amora/build/tmp/work/arm1176jzfshf-vfp-poky-linux-musleabi/rpi-libcamera-apps/git-r0/git/core/video_options.hpp:14:
| In file included from /home/yoctouser/amora/build/tmp/work/arm1176jzfshf-vfp-poky-linux-musleabi/rpi-libcamera-apps/git-r0/git/core/options.hpp:21:
| In file included from /home/yoctouser/amora/build/tmp/work/arm1176jzfshf-vfp-poky-linux-musleabi/rpi-libcamera-apps/git-r0/git/core/logging.hpp:1:
| In file included from /home/yoctouser/amora/build/tmp/work/arm1176jzfshf-vfp-poky-linux-musleabi/rpi-libcamera-apps/git-r0/git/core/libcamera_app.hpp:34:
| /home/yoctouser/amora/build/tmp/work/arm1176jzfshf-vfp-poky-linux-musleabi/rpi-libcamera-apps/git-r0/git/core/stream_info.hpp:11:10: fatal error: 'libcamera/color_space.h' file not found
| #include <libcamera/color_space.h>
|          ^~~~~~~~~~~~~~~~~~~~~~~~~
| 1 error generated.
```

Reply:
Submitted PR upstream - https://github.com/openembedded/meta-openembedded/pull/619.

Reply:
We'll have to hold off on merging this PR until upstream PR is completed.

Reply:
Sounds good. I don't think they accept PRs on GitHub. You'll probably have to chase it over the mailing list.

Reply:
Submitted patch via email - https://lists.openembedded.org/g/openembedded-devel/message/99061.

Reply:
This is now available upstream - https://github.com/openembedded/meta-openembedded/commit/9ab32b214b54a58baf49a725dddf1c720d4195d1.